bids/layout/layout.py

```diff
@@ -567,7 +567,7 @@ def to_df(self, metadata=False, **filters):
         query = self.session.query(Tag).filter(Tag.file_path.in_(file_paths))
 
         if not metadata:
-            query = query.join(Entity).filter(Tag.is_metadata == False)
```

Comment:
I think you need to revert these. These aren't bools but ORM objects, so `== False/True` actually is distinct from `is False/True`.

Reply:
Isn't the difference result between `is False` and `== False` a bug by itself?

Reply:
No. It's an API choice of sqlalchemy to use `__eq__()` instead of `equals()` to construct their queries.

Reply:
I understand:
* `x is False` is equivalent to `id(x) == id(False)`
* `x == False` is equivalent to `x == 0`

Should `__id__` be redefined for ORM objects? That would be strange. I guess we could write `bool(x) is False`. I cannot recall which linter suggested this change, but it clearly doesn't make sense.